electricitymap/capacity_parsers/EMBER.py

```diff
@@ -0,0 +1,176 @@
+import io
+from datetime import datetime
+from logging import getLogger
+
+import pandas as pd
+import pycountry
+from bs4 import BeautifulSoup
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+
+""" Collects capacity data from the yearly electricity data from Ember. The data and documentation can be found here: https://ember-climate.org/data-catalogue/yearly-electricity-data/"""
+logger = getLogger(__name__)
+EMBER_VARIABLE_TO_MODE = {
+    "Bioenergy": "biomass",
+    "Coal": "coal",
+    "Gas": "gas",
+    "Hydro": "hydro",
+    "Nuclear": "nuclear",
+    "Other Fossil": "unknown",  # mostly oil it seems
+    "Other Renewables": "unknown",
+    "Solar": "solar",
+    "Wind": "wind",
+}
+
+SPECIFIC_MODE_MAPPING = {
+    "BD": {"Other Fossil": "oil"},
+    "CO": {"Other Fossil": "oil"},
+    "CY": {"Other Fossil": "oil"},
+    "KR": {"Other Fossil": "oil"},
+    "KW": {"Other Fossil": "oil"},
+    "MN": {"Other Fossil": "coal"},
+    "SG": {"Other Fossil": "coal"},
+    "SV": {"Other Fossil": "oil"},
+    "TR": {"Other Fossil": "oil", "Other Renewables": "geothermal"},
+    "TW": {"Other Fossil": "oil"},
+    "ZA": {"Other Fossil": "oil"},
+}
+
+EMBER_ZONES = [
```

Comment:
Didn't we also talked about making this part of the yaml config ?

Reply:
yes we did but i've not updated any config yet, this is just a way of keeping track of which zones are updated using this parser